USE_CASES.md

```diff
@@ -127,3 +128,291 @@ Find more information about all of SendGrid's whitelabeling related documentatio
+
+In your `Gemfile` add `gem figaro` and run `$ bundle update`
+
+Generate an `application.yml` file, which will be used to map environment variables to their values:
+
+```
+$ figaro install
+```
+
+Make sure that `config/application.yml` has been added to your `.gitignore` file,
+
+#14
+
+Open `config/application.yml` and add:
+
+Note: Replace the sample values with yours, the **values are NOT added as STRINGS**
+
+```ruby
+#TODO:
+SENDGRID_USERNAME: app02268941@heroku.com
+SENDGRID_PASSWORD: 5qapohdfq1012
+SENDGRID_API_KEY: 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
+```
+
+#15
+
+Retrieve your SendGrid Username and Password and add them to `config/application.yml`:
+
+```
+$ heroku config:get SENDGRID_USERNAME
+$ heroku config:get SENDGRID_PASSWORD
+```
+
+#16
+
+In your app's dashboard in Heroku, on the upper left side you will see the SendGrid icon, click on the link, go to _Settings_, then _API Keys_ and generate an API key for your app. Add this to `config/application.yml`.
+
+You will also need to add your API Key as a production variable to Heroku:
+
+```
+$ heroku config:add SENDGRID_API_KEY="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"
+```
+
+### SETUP MAILER
+
+#17
+
+Create a file in `config/initializers` named `setup_mail.rb`:
+
+```
+$ touch config/initializers/setup_mail.rb
+```
+add the following code:
+
+```ruby
+if Rails.env.development? || Rails.env.production?
+  ActionMailer::Base.delivery_method = :smtp
+  ActionMailer::Base.smtp_settings = {
+    address:        'smtp.sendgrid.net',
+    port:           '587',
+    authentication: :plain,
+    user_name:      ENV['SENDGRID_USERNAME'],
+    password:       ENV['SENDGRID_PASSWORD'],
+    domain:         'heroku.com',
+    enable_starttls_auto: true
+  }
+end
+```
+The code in `config/initialize` runs when our app starts. We need to configure some special settings to send emails.
+
+#18
+
+Lets, generate a mailer
+
+```ruby
+$ rails generate mailer HelloMailer
+```
+
+#19
+
+In your `Gemfile` add `gem sendgrid-ruby` and run `$ bundle update`
+
+#20
+
+In `app/mailers/hello_mailer.rb` add:
+
+```ruby
+require 'sendgrid-ruby'
+
+class HelloMailer < ApplicationMailer
+  include SendGrid
+
+  def self.new_message(email)
+
+    from = Email.new(email: 'replacethiswithyouremail@gmail.com') #TODO
+    subject = 'Hello World from the SendGrid Ruby Library!'
+    to = Email.new(email: email)
+    content = Content.new(type: 'text/plain', value: 'Hello, Email!')
+    mail = Mail.new(from, subject, to, content)
+
+    sg = SendGrid::API.new(api_key: ENV['SENDGRID_API_KEY'])
+    response = sg.client.mail._('send').post(request_body: mail.to_json)
+    puts response.status_code
+    puts response.body
+    puts response.headers
+
+  end
+end
+```
+
+This is a simple example, but if you would like to to use a SendGrid Template, have a look at this [example](https://github.com/sendgrid/sendgrid-ruby/blob/master/USE_CASES.md)
```

Comment:
Similar to before, instead of having a short one-word link, let's try to make the link easier to click by making it larger. Instead of:

> have a look at this [example]

This would be better as:

    [have a look at this example]